src/sentry/features/__init__.py

```diff
@@ -112,8 +112,10 @@
 default_manager.add("organizations:org-subdomains", OrganizationFeature)
 default_manager.add("organizations:performance-anomaly-detection-ui", OrganizationFeature, True)
 default_manager.add("organizations:performance-chart-interpolation", OrganizationFeature, True)
-default_manager.add("organizations:performance-dry-run-mep", OrganizationFeature, True)
-default_manager.add("organizations:performance-n-plus-one-api-calls-detector", OrganizationFeature)
+default_manager.add("organizations:performance-frontend-use-events-endpoint", OrganizationFeature, True)
+default_manager.add("organizations:performance-issues", OrganizationFeature, True)
+default_manager.add("organizations:performance-issues-ingest", OrganizationFeature)
+default_manager.add("organizations:performance-issues-post-process-group", OrganizationFeature)
```

Comment:
These look like they were added by mistake, I expected to only see the removal of the dry run flag

Reply:
They were good catch